charts/rapid-response/README.md

```diff
@@ -48,6 +48,9 @@ The following table lists the configurable parameters of the Sysdig Rapid Respon
 | `rapidResponse.sslVerifyCertificate`      | Set it to `false` for disabling the certificate verification                            | `true`                                                        |
 | `rapidResponse.tolerations`               | The tolerations for scheduling	                                                      | `node-role.kubernetes.io/master:NoSchedule`                   |
 |                                                                                                                                       `node-role.kubernetes.io/control-plane:NoSchedule`            |
+| `rbac.create`                             | If true, create & use RBAC resources                                                    | `true`   |
+| `serviceAccount.create`                   | Create serviceAccount                                                                   | `true`   |
+| `serviceAccount.name`                     | Use this value as serviceAccountName                                                    | ` `   |
```

Comment:
This one can be removed from the readme since we are not going to deploy any rbac/serviceaccount on customer clusters